moped-editor/src/queries/subprojects.js

```diff
@@ -1,24 +1,25 @@
 import { gql } from "@apollo/client";
 
-
 export const SUBPROJECT_QUERY = gql`
   query SubprojectSummary($projectId: Int) {
-    subprojects: moped_project(where:{project_id: { _eq: $projectId } }) {
+    subprojects: moped_project(where: { project_id: { _eq: $projectId } }) {
       moped_projects {
         project_name
         project_id
         status_id
         current_phase
       }
     }
-    subprojectOptions: moped_project(where: {
-      _and: [
-        {is_deleted: {_eq: false}},
-        {parent_project_id: {_is_null: true}}
-        {project_id: {_neq: $projectId}}
-      ],
-      _not: {moped_projects: {}}
-    }) {
+    subprojectOptions: moped_project(
+      where: {
+        _and: [
+          { is_deleted: { _eq: false } }
+          { parent_project_id: { _is_null: true } }
+          { project_id: { _neq: $projectId } }
+        ]
+        _not: { moped_projects: {} }
+      }
+    ) {
```

Comment:
observing that this is merely linting 👍

Reply:
...also observing that the `subprojectOptions` typename aliasing is cool and i forgot that it was possible/necessary
https://hasura.io/blog/customising-alias-graphql-fields-with-hasura/